crates/autopilot/src/boundary/mod.rs

```diff
@@ -43,3 +43,9 @@ pub fn buffered_web3_client(ethrpc: &Url) -> Web3 {
         });
     shared::ethrpc::web3(&ethrpc_args, &http_factory, ethrpc, "base")
 }
+
+pub struct SolvableOrders {
+    pub orders: Vec<model::order::Order>,
+    pub quotes: HashMap<crate::domain::OrderUid, Quote>,
```

Comment:
ubernit: can we name this as
`pub quotes: HashMap<domain::OrderUid, domain::Quote>,` 

just so that we are consistent with how we refer to domain from other modules.